client/app/app.js

```diff
@@ -155,4 +155,55 @@
     }])
   //---------------------------------------------------------
 
+   app.controller('walletCtrl', ['$scope',
```

Comment:
This code needs to be moved into its own module folder.
Reach out if you would like another walk through on how to do this.